src/question/editor.rs

```diff
@@ -34,17 +34,20 @@ impl<'a> Default for Editor<'a> {
 }
 
 fn get_editor() -> Command {
-    Command::new(
-        env::var_os("VISUAL")
-            .or_else(|| env::var_os("EDITOR"))
-            .unwrap_or_else(|| {
-                if cfg!(windows) {
-                    "notepad".into()
-                } else {
-                    "vim".into()
-                }
-            }),
-    )
+    let editor_command = env::var_os("VISUAL")
+        .or_else(|| env::var_os("EDITOR"))
+        .unwrap_or_else(|| {
+            if cfg!(windows) {
+                "notepad".into()
+            } else {
+                "vim".into()
+            }
+        });
+
+    let parts = shell_words::split(editor_command.to_str().unwrap()).unwrap();
```

Comment:
I don't think these unwraps are justified. In my opinion, if any of these error the default editor should be used instead of panicking. For example:

```rust
    let editor_args = env::var_os("VISUAL")
        .or_else(|| env::var_os("EDITOR"))
        .map(|editor_command| editor_command.to_str().ok())
        .flatten()
        .map(|editor_command| shell_words::split(editor_command).ok()) // These should change
        .flatten()                                                     // based on cfg()
        .unwrap_or_else(|| {
            if cfg!(windows) {
                vec!["notepad".into()]
            } else {
                vec!["vim".into()]
            }
        });
```